engine/src/main/java/org/terasology/engine/rendering/nui/layers/mainMenu/UniverseSetupScreen.java

```diff
@@ -357,8 +357,8 @@ private int findIndex(List<WorldSetupWrapper> worldsList, String worldName) {
 
     private void initAssets() {
 
-        ModuleEnvironment environment = context.get(ModuleManager.class).getEnvironment();
-        BlockFamilyLibrary library =  new BlockFamilyLibrary(environment, context);
+        ModuleEnvironment env = context.get(ModuleManager.class).getEnvironment();
```

Comment:
We have the `environment` as class member, but it is only written in `setEnvironment`. 

It feels wrong to me that `initAssets` resolves the environment again, or, in case it gets called first, does not initialize the environment. To make this even more weird, we also have the `moduleManager` as class member, yet resolve it here again via the context :see_no_evil: